.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 
 #### Prerequisites checklist
 
-- [ ] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).
+- [ ] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/main/CONTRIBUTING.md).
```

Comment:
```suggestion
- [ ] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).
```
this way it will always work, now and later, and won't need to be coordinated with the default branch change.

Same for every URL in the PR.

Reply:
Thanks for the review @ljharb. I made the changes in all the URLs.